.evergreen/config.yml

```diff
@@ -94,38 +94,45 @@ functions:
           go version
           go env
 
+          LIBMONGOCRYPT_TAG="1.5.0-rc2"
+          # LIBMONGOCRYPT_COMMIT is the commit on libmongocrypt for the tag LIBMONGOCRYPT_TAG.
+          LIBMONGOCRYPT_COMMIT="1756c5120814152c15a2eee7a679a33a4764ba63"
           # Install libmongocrypt based on OS.
           if [ "Windows_NT" = "$OS" ]; then
              mkdir -p c:/libmongocrypt/include
              mkdir -p c:/libmongocrypt/bin
              # TODO (GODRIVER-2436): do not use pre-release of libmongocrypt.
```

Comment:
I suppose `1.5.0-rc2` is still a "prerelease"? So, we should keep this `TODO` and not resolve GODRIVER-2436?

Reply:
Correct. `1.5.0-rc2` is a prerelease. GODRIVER-2436 can be resolved once 1.5.0 stable is released.